src/platform/Platform.js

```diff
@@ -135,11 +140,16 @@ Platform.prototype.createUrl = function(path, options) {
     options = options || {};
 
     var builtUrl = '',
-        hasHttp = path.indexOf('http://') != -1 || path.indexOf('https://') != -1;
+        hasHttp = path.indexOf('http://') != -1 || path.indexOf('https://') != -1,
+        alreadyPrefixed = this._knownPrefixes.some(function(prefix) {
+            return path.indexOf(prefix) === 0;
+        });
 
     if (options.addServer && !hasHttp) builtUrl += this._server;
 
-    if (path.indexOf(Platform._urlPrefix) == -1 && !hasHttp) builtUrl += Platform._urlPrefix + '/' + Platform._apiVersion;
+    if (path.indexOf(Platform._urlPrefix) == -1 && !hasHttp && !alreadyPrefixed) {
+        builtUrl += [Platform._urlPrefix, Platform._apiVersion].join('/');
```

Comment:
Weird code style to add a slash between two strings )

Reply:
It's just more efficient than string concatenation. Anyway, i can replace it.